src/server/common/test-helpers/component-helpers.js

```diff
@@ -6,13 +6,13 @@ import nunjucks from 'nunjucks'
 import { load } from 'cheerio'
 import { camelCase, upperFirst } from 'lodash'
 
-import * as filters from '../../../config/nunjucks/filters/filters.js'
-import * as globals from '../../../config/nunjucks/globals/globals.js'
+import * as filters from '#config/nunjucks/filters/filters.js'
+import * as globals from '#config/nunjucks/globals/globals.js'
 
 const dirname = path.dirname(fileURLToPath(import.meta.url))
 const nunjucksTestEnv = nunjucks.configure(
   [
-    '../../../../node_modules/govuk-frontend/dist/',
+    'node_modules/govuk-frontend/dist/',
```

Comment:
LGTM except I have no idea about this part, so I'll defer to a Nunjucks black belt!